Akanksha's intro blog for Okta (#1572)
oktadeveloper/okta-blog: _source/_posts/2025-04-18-akanksha-bhasin-intro.md

---
layout: blog_post
title: "From Astronomy Geek to Oktanaut: Landing as a Dev Advocate at Okta"
author: akanksha-bhasin
by: advocate
communities: [javascript,security,java,python]
description: "Akanksha Bhasin joins the Okta team!"
tags: [javascript, python, advocacy, okta]
image: blog/akanksha-bhasin-intro/akanksha-bhasin-social-image.jpg
type: awareness
---

> I'm Akanksha Bhasin, and I'm all about building community for developers. I come from a developer background, and for over six years, I've been building and growing communities for developers. On top of that, I've spent the last four years diving into DevRel. I've had the chance to work with some fast-growing companies, helping them build their communities worldwide. That means getting the word out about emerging tech to developers, organizing significant events and hackathons, speaking at conferences, fostering partnerships, and supporting open-source projects.

Reviewer: ```suggestion
I'm Akanksha Bhasin, and I'm all about building community for developers. I come from a developer background, and for over six years, I've been building and growing communities for developers. On top of that, I've spent the last four years diving into Developer Relations. I've had the chance to work with some fast-growing companies, helping them build their communities worldwide. That means getting the word out about emerging tech to developers, organizing significant events and hackathons, speaking at conferences, fostering partnerships, and supporting open-source projects.
```

My lifelong curiosity, from early science experiments and a fascination with astronomy,  has propelled me into an exciting new orbit: officially becoming an Oktanaut at Okta\! 🚀🌠🧑‍🚀

## My journey into the tech world 

My tech journey has been quite a ride\! I began by experimenting with various things, including Arduino, different aspects of programming, and building AI applications.

> What piqued my interest wasn't just web development and the scope of AI applications. Back in 2018, when Google Assistant came out, I was fascinated by how voice could fit into our everyday lives, from controlling appliances to using apps, all with your voice\! So, I started building voice AI applications for Google Assistant and Alexa and eventually began teaching what I learned at professional meetups and colleges all over India. I got featured at Google I/O for my work in Voice AI. 

Reviewer: ```suggestion
What piqued my interest wasn't just web development and the scope of AI applications. Back in 2018, when Google Assistant came out, I was fascinated by how voice could fit into our everyday lives, from controlling appliances to using apps, all with your voice\! So, I started building Voice AI applications for Google Assistant and Alexa and eventually began teaching what I learned at professional meetups and colleges all over India. Which then led to being featured at Google I/O for my work in Voice AI. 
```

{% img blog/akanksha-bhasin-intro/google-io.jpeg alt:"Photo of Akanksha featured at Google I/O" width:"800" %}{: .center-image }

## Building developer communities: my passion and journey

> Driven by a passion for community building that began with Voice AI technology, I later founded Women in Voice, India, to empower more women to build careers in tech. Alongside this endeavor, I've delivered sessions worldwide, hosted hackathons and events, led Google Developer Groups on my campus, and volunteered to organize the largest Devfests in my region, which gathered 3,000+ developers.

Reviewer: ```suggestion
Driven by a passion for community building that began with Voice AI technology, I later founded Women in Voice, India, to empower more women to build careers in tech. Alongside this endeavor, I've delivered sessions worldwide, hosted hackathons and events, led Google Developer Groups on my campus, and volunteered to organize the largest DevFests in my region, which gathered 3,000+ developers.
```

Reviewer: Were these multiple DevFests or just one big one?

Author: This one was really huge, but we hosted this yearly.

> My fondest memory has been in Mumbai, where I got to teach how to build AI applications in areas without much tech awareness and opportunities. Seeing someone's face light up when they finally understood new technology was priceless and kept me going. Life should always be about giving back. Even if you have impacted just one person's life, you have done something priceless. 

Reviewer: ```suggestion
My fondest memory has been in Mumbai, where I got to teach how to build AI applications in areas without much tech awareness and opportunities. Seeing someone's face light up when they finally understood new technology was priceless and kept me going. Life is about giving back. Even if you have impacted just one person's life, you have done something priceless. 
```

{% img blog/akanksha-bhasin-intro/wiv-award.jpeg alt:"Akanksha won the Women in Voice award" width:"800" %}{: .center-image }

## Embracing developer advocacy at Okta

I'm so excited to join the Okta team; it's a pinch-me moment\! I love our principle of "Building a world where identity belongs to you\!" Innovation is thriving, and change is here, making this the opportune and crucial moment for security. 

> Identity and security are the shields and the first line of defense for any software. They wrap every API, CLI, and SaaS workload in a magnetic field of strong authentication and privileged access. Therefore, a compromised credential can topple an entire software system, just like a small puncture in a spacesuit; the whole mission can fail. With that, security cannot remain a checklist item; instead, security is the foundational layer of every enterprise application. I want to be part of this mission to help independent software vendors (ISVs) and developers secure user identities in their applications\! 

Reviewer: ```suggestion
Identity security and access management are the shields and the first line of defense for any software. They wrap every API, CLI, and SaaS workload in a magnetic field of strong authentication and privileged access. Without it, a compromised credential can topple an entire software system, just like a small puncture in a spacesuit; the whole mission can fail. With that, security can't remain a checklist item; instead, security is the foundational layer of every enterprise application. I want to be part of this mission to help independent software vendors (ISVs) and developers secure user identities in their applications\! 
```

## Let's connect\! 

> When I'm not geeking out about tech, you'll probably find me dancing or exploring new places and cultures. If you see me with my phone out, I'm trying to capture a beautiful sky. These moments just make me feel calm and happy. Connect with me on [LinkedIn](https://linkedin.com/in/akankshabhasin); I'd love to chat with you\!

Reviewer: ```suggestion
When I'm not geeking out about tech, you'll probably find me dancing or exploring new places and cultures. If you catch me outside with my phone, I'm probably trying to capture a beautiful sky. These simple moments make me feel calm and happy. Connect with me on [LinkedIn](https://linkedin.com/in/akankshabhasin); I'd love to chat with you\!
```
